### PR TITLE
GAWB-467 Enable unit testing with MySQL

### PR DESCRIPTION
**Pull request by jmthibault79**
First pass at MySQL unit testing, has some rough edges.  ~25 tests fail and fixing them is tracked under GAWB-491 

Quickstart, from the README updates to this PR:

```
$ docker build -t broadinstitute/rawls:dev-base ${RAWLS_SRC_DIR}
$ docker build -t broadinstitute/rawls:test-mysql ${RAWLS_SRC_DIR}/docker/test-mysql
$ docker run -d --name rawls-test -v ${RAWLS_SRC_DIR}:/app:rw broadinstitute/rawls:test-mysql
$ docker exec -it rawls-test bash
```

Then inside the Docker container:

```
# cd /app
# sbt -Dtestdb=mysql test
```
- [x] **Submitter**: Rebase to develop. DO NOT SQUASH
- [x] **Submitter**: Make sure Swagger is updated if API changes
- [x] **Submitter**: Make sure documentation for code is complete
- [x] **Submitter**: Make sure liquibase is updated if appropriate
- [x] **Submitter**: Review code comments; remove done TODOs, create stories for remaining TODOs
- [x] **Submitter**: Include the JIRA issue number in the PR description
- [x] **Submitter**: Add description or comments on the PR explaining the hows/whys (if not obvious)
- [x] Tell  that the PR exists if he wants to look at it
- [x] Anoint a lead reviewer (LR). **Assign PR to LR**
- [x] **LR**: Initial review by LR and others.
- [x] Comment / review / update cycle:
  - Rest of team may comments on PR at will
  - **LR assigns to submitter** for feedback fixes
  - Submitter updates documentation as needed
  - Submitter rebases to develop again if necessary
  - Submitter makes further commits. DO NOT SQUASH. **Reassign to LR** for further feedback
- [x]  sign off
- [x] **LR** sign off
- [x] **Assign to submitter** to finalize
- [x] **Submitter**: Squash commits, rebase if necessary
- [x] **Submitter**: Verify all tests go green, including CI tests
- [x] **Submitter**: Merge to develop 
- [x] **Submitter**: Delete branch after merge
- [x] **Submitter**: Check configuration files in Jenkins in case they need changes
- [x] **Submitter**: **Test this change works on dev environment after deployment**. YOU own getting it fixed if dev isn't working for ANY reason!
- [x] **Submitter**: Verify swagger UI on dev environment still works after deployment
- [x] **Submitter**: Inform other teams of any API changes via hipchat and/or email
- [x] **Submitter**: Mark JIRA issue as resolved once this checklist is completed

### CODE REVIEW

File: docker/install.sh [hunk 1]
```diff
@@ -9,6 +9,9 @@ if [ -e "jenkins_env.sh" ]; then
 	source "jenkins_env.sh"
 fi
 
+# catch sbt issues separately
+sbt update && echo "sbt updated successefully."
```
Comment (abaumann) [inline docker/install.sh:13]: I believe you mean successéfully

Comment (jmthibault79) [inline docker/install.sh:13]: o, suite successe

File: build.sbt [hunk 2]
```diff
@@ -52,7 +47,12 @@ libraryDependencies ++= {
     "com.typesafe.slick" %% "slick-hikaricp" % slickV,
     "mysql" % "mysql-connector-java" % "5.1.38",
     "org.liquibase" % "liquibase-core" % "3.3.5",
-    "ch.qos.logback" % "logback-classic" % "1.1.6"
+    "ch.qos.logback" % "logback-classic" % "1.1.6",
+    "com.typesafe.akka" %% "akka-testkit" % akkaV % "test",
+    "io.spray" %% "spray-testkit" % sprayV % "test",
+    "org.scalatest" %% "scalatest" % "2.2.4" % "test",
+    "org.mock-server" % "mockserver-netty" % "3.9.2" % "test",
+    "com.h2database" % "h2" % "1.4.191" % "test"
```
Comment (dvoet) [inline build.sbt:55]: was there a decision to keep h2?

Comment (jmthibault79) [inline build.sbt:55]: Yes.  For this iteration, I don't want to make running Docker and MySQL mandatory for unit test.

**Comment by dvoet** [comment]
:+1:

**Pull request merged**